app/src/main/java/de/danoeh/antennapod/ui/screen/queue/QueueFragment.java

```diff
@@ -516,6 +517,14 @@ private void loadItems(final boolean restoreScrollPosition) {
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribe(items -> {
                     queue = items;
+                    if (queue.isEmpty() && DBReader.getTotalEpisodeCount(new FeedItemFilter(FeedItemFilter.NEW)) > 0) {
+                        emptyView.setMessage(R.string.no_queue_items_inbox_has_items_label);
+                        emptyView.setButtonText(R.string.no_queue_items_inbox_has_items_button_label);
+                        emptyView.setButtonVisibility(View.VISIBLE);
+                        emptyView.setButtonOnClickListener(v -> {
+                            ((MainActivity) getActivity()).loadChildFragment(new InboxFragment());
+                        });
```

Comment:
The brackets can be removed here, I think Android Studio probably also recommends this